tlcc/tlcc.go

```diff
@@ -64,6 +64,7 @@ func (t *TrustedLedgerCC) getLocalAttestationReport(stub shim.ChaincodeStubInter
 
 	reportAsBytes, enclavePk, err := t.enclave.GetLocalAttestationReport([]byte(targetInfo))
 	if err != nil {
+		logger.Errorf("t.enclave.GetLocalAttestationReport failed: %s", err)
```

Comment:
why adding this logging statement here? 
- we return the error already?
- having two different error messages is wrong, better create error msg, then log and return

Reply:
* While trying to debug these parts, it was (very!!!) convenient to have the error message in `peer.err`.
* I agree we can avoid msg duplication, fixed it!